javacutil/src/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -336,8 +337,11 @@ public static Tree getAssignmentContext(final TreePath treePath) {
                 (node instanceof NewArrayTree) ||
                 (node instanceof NewClassTree) ||
                 (node instanceof ReturnTree) ||
-                (node instanceof VariableTree))
+                (node instanceof VariableTree)) {
             return node;
+        } else if (node instanceof ConditionalExpressionTree) {
```

Comment:
No, that would return incorrect assignment contexts.  For example:
````
String s = m.toString();
```
Calling this method on the expression `m`  should not return `s`'s tree. 
I've improved the documentation to make this clear.